lib/role.rb

```diff
@@ -0,0 +1,38 @@
+class Role 
+  attr_accessor :role_type, :proposal, :user
+  def initialize(user,proposal)
+    self.user = user
+    self.proposal = proposal
```

Comment:
I'm not sure it's helpful to carry around the user or proposal after the initialization?